jetstream_pt/layers.py

```diff
@@ -367,9 +367,25 @@ def apply_rotary_emb(
 def repeat_kv(x: torch.Tensor, n_rep: int) -> torch.Tensor:
   """torch.repeat_interleave(x, dim=2, repeats=n_rep)."""
 
-  bs, n_kv_heads, slen, head_dim = x.shape
+  bs, n_kv_heads, slen, head_dim = (
```

Comment:
bs, n_kv_heads, slen, head_dim, *_ = x.shape

Reply:
I see, should be *_, bs, n_kv_heads, slen, head_dim.= x.shape ?